hub/policy/src/main/java/uk/gov/ida/hub/policy/logging/HubEventLogger.java

```diff
@@ -215,7 +219,8 @@ public void logIdpAuthnSucceededEvent(SessionId sessionId,
         details.put(idp_entity_id, idpEntityId);
         details.put(pid, persistentId.getNameId());
         details.put(minimum_level_of_assurance, minimumLevelOfAssurance.name());
-        details.put(required_level_of_assurance, requiredLevelOfAssurance.name());
+        details.put(maximum_level_of_assurance, maximumLevelOfAssurance.name());
```

Comment:
I don't think I particularly understand why maximum_level_of_assurance would ever be a thing. We'll take any level of assurance, as high as exists in our scheme, for any transaction. I'm struggling to think of a case when we wouldn't.